eng/release/Scripts/GetReleaseVersion.ps1

```diff
@@ -1,30 +1,47 @@
 [CmdletBinding()]
 Param(
     [Parameter(Mandatory=$true)][string] $BarId,
-    [Parameter(Mandatory=$true)][string] $MaestroToken,
-    [Parameter(Mandatory=$false)][string] $MaestroApiEndPoint = 'https://maestro-prod.westus2.cloudapp.azure.com',
-    [Parameter(Mandatory=$false)][string] $MaestroApiVersion = '2020-02-20',
+    [Parameter(Mandatory=$false)][string] $MaestroApiEndPoint = 'https://maestro.dot.net',
     [Parameter(Mandatory=$false)][string] $TaskVariableName = $null,
+    [Parameter(Mandatory=$false)][string] $DarcVersion = $null,
     [Parameter(Mandatory=$false)][switch] $IncludeV
 )
 
 $ErrorActionPreference = 'Stop'
 Set-StrictMode -Version 2.0
 
-[array]$releaseData = Invoke-RestMethod `
-    -Uri "$MaestroApiEndPoint/api/assets?buildId=$BarId&name=dotnet-monitor&api-version=$MaestroApiVersion" `
-    -Method 'GET' `
-    -Headers @{ 'accept' = 'application/json'; 'Authorization' = "Bearer $MaestroToken" }
+$ci = $true
+$darc = $null
+try {
+    $darc = (Get-Command darc).Source
+}
+catch {
+    . $PSScriptRoot\..\..\common\tools.ps1
+    $darc = Get-Darc $DarcVersion
+}
```

Comment:
Did you want to create a common function between the two scripts? It seems like there's a lot of overlap.